owner/manager_test.go

```diff
@@ -56,12 +56,12 @@ func TestSingle(t *testing.T) {
 	}
 	store, err := mockstore.NewMockStore()
 	if err != nil {
-		t.Fatal(err)
+		require.NoError(t, err)
 	}
```

Comment:
```
if err != nil {
 Fatal...
}
```

is equivalent to

```
require.NoError(t, err)
```

There is no need to guard the require assertion with an if branch.